google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/CloudSdkService.java

```diff
@@ -27,7 +29,7 @@ public static CloudSdkService getInstance() {
     return ServiceManager.getService(CloudSdkService.class);
   }
 
-  public abstract String getCloudSdkHomePath();
+  public abstract Path getCloudSdkHomePath();
 
   public abstract void setCloudSdkHomePath(String path);
```

Comment:
I thought about this, but we're really only using the String part of the Path to set the value in propertiesComponent.
In practice, we'd be converting to a Path to call the method, and then the method would do the opposite conversion to String.

That being said, which would you prefer? String or Path?